index.js

```diff
@@ -194,25 +207,86 @@ class Timber {
 	}
 }
 
-module.exports = new Timber({
+const logger = new Timber({
 	name: is.main ? 'main' : null
 });
 
-if (is.main) {
-	const rendererLogger = new Timber({name: 'renderer'});
-	if (electron.ipcMain.listenerCount(logChannel) === 0) {
-		electron.ipcMain.on(logChannel, (event, data) => {
-			rendererLogger.log(...data);
-		});
+logger.hookConsole = ({main, renderer} = {main: is.main, renderer: is.renderer}) => {
+	if (main && is.renderer) {
+		throw new Error('You cannot hook the console in the main process from a renderer process.');
 	}
-	if (electron.ipcMain.listenerCount(warnChannel) === 0) {
-		electron.ipcMain.on(warnChannel, (event, data) => {
-			rendererLogger.warn(...data);
-		});
+
+	const hookThisConsole = (is.main && main) || (is.renderer && renderer);
+	const shouldHookRenderers = is.main && renderer;
+
+	if (hookThisConsole) {
+		if (isConsoleHooked) {
+			throw new Error('Console already hooked!');
```

Comment:
When/if `hookConsole()` is implemented, should it be a noop rather than an error if it's called and it's already hooked?

```js
if (isConsoleHooked) {
  return;
}
```

Reply:
I think it should just be a noop.

Reply:
Giving this some thought, I've decided that I don't like the idea of returning a noop or an unhook function.

I think it's probably better to keep things consistent so it all works according to expectations, etc. So I changed it to return an unhook function that reflects the arguments you passed in. 

Eg: 
```js
// Returns an unhook function that unhooks main but not renderer.
logger.hookConsole({ main: true, renderer: false });
// Returns an unhook function that unhooks main and renderer.
logger.hookConsole({ main: true, renderer: true });
 // etc
```